### PR TITLE
added poster judges

### PR DESCRIPTION
**Pull request by chrismpr86**

### CODE REVIEW

**Review by rowlandm — CHANGES_REQUESTED** [review]
All the new things talk about the regular organising committee....

File: volunteering.md [hunk 1]
```diff
@@ -21,6 +21,24 @@ The expectations of the co-chair are:
 9. To ensure that the Diversity, Equity, and Inclusion of the event is up to the highest standard possible, given the resources and time available
 
 
+# Expectations of poster judges
+
+The expectations of the regular organising committee members are:
```
Comment (rowlandm) [inline volunteering.md:26]: This is for poster judges

File: volunteering.md [hunk 2]
```diff
@@ -21,6 +21,24 @@ The expectations of the co-chair are:
...
+1. To represent the wider community of RSEs
+2. To review each poster and ask questions if needed
+3. To identify outstanding posters from the event
+
+# Expectations of the observers
+
+The expectations of the regular organising committee members are:
```
Comment (rowlandm) [inline volunteering.md:33]: This is for observers. This was a mistake from before.

**Review by rowlandm — COMMENTED** [review]
Can the secretary be before the others please?

And where is the session chair expectations? I couldn't find them.

**Comment by chrismpr86** [comment]
you can review now, thanks :)

**Review by rowlandm — APPROVED** [review]

**Pull request merged**